build/docs.js

```diff
@@ -6,8 +6,9 @@ const { glob } = require("glob");
 const fm = require("front-matter");
 const mk = require("marked");
 
-async function buildDocs() {
-  glob(path.resolve(__dirname, "../src/docs/**/*.md"), async (err, files) => {
+async function mdToJson(mdDir, jsonFile) {
+  mdDir = path.join(mdDir)
```

Comment:
Why not just specify the full path in the call to `mdToJson` rather than join paths later?